source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/files/product-artifact.tsx

```diff
@@ -56,15 +58,30 @@ class ProductArtifact extends React.Component<IProps> {
             {this.humanReadableName()}
           </span>
         </div>
-        <div className='list'>
+        <div>
           <EmptyLabel
-            className='m-l-lg m-b-lg'
+            className='m-t-lg m-l-lg m-b-lg'
             condition={!isEmpty(artifacts)}
-            label={'No artifacts'}
+            label={t('project.products.noArtifacts')}
           >
+            <div className='flex p-l-md p-t-sm p-b-sm p-r-md gray-text bold'>
+              <div className='flex-70'>
+                <span>{t('project.products.filename')}</span>
+              </div>
+              <div className='flex flex-30'>
+                <div className='flex-grow'>
+                  <span>{t('project.products.updated')}</span>
+                </div>
+                <div className='flex-30 text-align-right'>
+                  <span className='m-r-md'>{t('project.products.size')}</span>
+                </div>
+                <div className='flex-10'></div>
+              </div>
+
+            </div>
             {
               artifacts.map((artifact, i) =>
-                <Artifact key={i} artifact={artifact} includeHeader={i === 0}/>
```

Comment:
yeah, this is much better.
I should have caught this previous logic as motivation for changing the header from it's position-absoluteing